INSTALL.md

```diff
@@ -53,18 +53,53 @@ sudo apt install -y dangerzone
 
 ### Fedora
 
-Add our repository following [these instructions](https://packagecloud.io/firstlookmedia/code/install#manual-rpm), or by running this script:
+### 1. Add our repository
+
+Start by installing our repository. This allows you to
+
+Open a terminal window and type the following commands:
 
 ```
-curl -s https://packagecloud.io/install/repositories/firstlookmedia/code/script.rpm.sh | sudo bash
+gpg --keyserver hkps://keys.openpgp.org --recv-key "DE28 AB24 1FA4 8260 FAC9  B8BA A7C9 B385 2260 4281"
+gpg --export --armor A7C9B38522604281 | sudo tee /etc/pki/rpm-gpg/RPM-GPG-dangerzone.pub > /dev/null
+sudo yum-config-manager --add-repo=https://packages.freedom.press/yum-tools-prod/dangerzone/dangerzone.repo
```

Comment:
Do we also have to `yum-config-manager --enable` the repo as well. If so, we can do it only if the downloaded repo file matches a known checksum, so that we are sure we add the proper repo.

Reply:
We may go with that approach. It's a long command, that I'd rather avoid, but it seems like the safest alternative (that doesn't need to user to do the fingerprint verification step -- but we're trusting github).

Reply:
I'll look at an alternative for importing directly to with `rpm --import` to import the fingerprint directly into Fedora's keyring.

Reply:
After discussion, we decided to go with the approach of [docker](https://docs.docker.com/engine/install/fedora/) and Tor and downloading a `.repo` file which references a remote key. And then we inform the user to verify the fingerprint matches the one on `INSTALL.md`.